spec.html

```diff
@@ -1145,6 +1145,26 @@ <h1>The String Type</h1>
         For historical reasons, it has significance to various algorithms.
       </p>
 
+      <emu-clause id="sec-ascii-case-sensitivity">
+        <h1>ASCII Case Insensitivity</h1>
+
+        <p>
+          The String values used to identify time zone identifiers, as well as many other identifiers specified in ECMA-402 such as locales, currencies, scripts, and calendars, as are interpreted in an ASCII-case-insensitive manner, treating the code units 0x0041 through 0x005A (corresponding to Unicode characters LATIN CAPITAL LETTER A through LATIN CAPITAL LETTER Z) as equivalent to the corresponding code units 0x0061 through 0x007A (corresponding to Unicode characters LATIN SMALL LETTER A through LATIN SMALL LETTER Z), both inclusive. No other case folding equivalences are applied.
```

Comment:
This spec text is migrated as-is from https://tc39.es/ecma402/#sec-case-sensitivity-and-case-mapping, with the only one change in the first sentence to clarify that in 262, only time zone identifiers are treated ASCII-case-insensitively.